docs/templates/nextflow/.gitignore

```diff
@@ -0,0 +1,3 @@
+.nextflow*
```

Comment:
I think this should maybe go in the main `.gitignore` file in the repository root directory

Reply:
`.nextflow*` is now in the root directory `.gitignore`